src/components/FeedPage/ListingsUserFeed.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Icon from '../../misc/Icon';
+import { Link as Listing } from 'react-router-dom';
+import EditListing from '../EditPage/EditListing';
+import PropTypes from 'prop-types';
+
+const StyledUserListings = styled.section``;
+
+const StyledListingWrapper = styled.div`
+  border: 2px solid #abc38e;
+  border-radius: 11px;
+  font-size: 15px;
+  padding: 5px;
+  align-items: center;
+  margin: 8px 0;
+  opacity: ${props => props.opacity};
+`;
+
+const StyledTitle = styled.h3`
+  padding: 0;
+  margin: 5px 0;
+`;
+
+const StyledShowIcon = styled(Listing)`
+  color: #abc38e;
+  font-size: 25px;
+  margin: 5px;
+  text-decoration: none;
+`;
+
+function ListingUserFeed({ listings, onDelete, history, onSaveChanges }) {
+  const [editMode, setEditMode] = useState(false);
+  const [editedListing, setEditedListing] = useState({});
+
+  function handleDeleteClick(id) {
+    onDelete(id);
+    history.push('/');
+  }
+
+  function handleEditClick(id) {
+    setEditMode(!editMode);
```

Comment:
Ich weiß, nicht, ob das voneinander abhängt, aber normalerweise steht const ganz oben in einer Funktion